packages/fetchai/contracts/erc1155/contract.py

```diff
@@ -26,7 +26,7 @@
 from aea_ledger_cosmos import CosmosApi
 from aea_ledger_ethereum import EthereumApi
 from aea_ledger_fetchai import FetchAIApi
-from google.protobuf.any_pb2 import Any as ProtoAny
+from google.protobuf.any_pb2 import Any as ProtoAny  # pylint: disable=E0611
```

Comment:
it's better to add pylint exclusion for `google.protobuf.any_pb2` in config
`ignored-modules`

Reply:
`google` is in the `ignored-modules` in the config (line 284 in `pyproject.toml`). Shouldn't that also include `google.protobuf.any_pb2`?

Reply:
Apparently not. Addressed.